plugins/languages/ltex-extra.nix

```diff
@@ -28,11 +28,12 @@ helpers.neovim-plugin.mkNeovimPlugin config {
       Whether to load dictionaries on startup.
     '';
 
-    load_langs = helpers.defaultNullOpts.mkNullable (types.listOf types.str) ''["en-US"]'' ''
+    load_langs = helpers.defaultNullOpts.mkListOf types.str [ "en-US" ] ''
       Languages for witch dicionnaries will be loaded.
       See `plugins.lsp.servers.ltex.languages` for possible values.
     '';
 
+    # FIXME consider using enum or nixvimTypes.logLevel
```

Comment:
We could indeed have an `enum` here.
However, using `nivimTypes.logLevel` is not possible as this option expects a string, not an int.

Reply:
Moved to #1711